frontend/src/components/my-links/RightPanel.tsx

```
@@ -0,0 +1,980 @@
+            <button 
+              disabled={selectedLinkIds.length === 0}
+              onClick={() => setIsMoveModalOpen(true)}
+              className="px-2.5 py-1.5 flex items-center gap-1.5 rounded-lg text-gray-600 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-700/50 disabled:opacity-40 disabled:hover:bg-transparent transition-colors text-[10px] font-medium"
+            >
+              <span className="material-symbols-outlined !text-[14px]">drive_file_move</span>
+              Move
+            </button>
+            <button
+              disabled={selectedLinkIds.length === 0}
+              onClick={() => {
+                selectedLinkIds.forEach((id) => deleteBookmarkMutation.mutate(id));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)